Ajout de pair:Group à Archipel

capucinemasson opened the pull request:
modified:   src/resources/Agent/Actor/Actor.js
	new file:   src/resources/Agent/Actor/Group/GroupEdit.js
	new file:   src/resources/Agent/Actor/Group/GroupList.js
	new file:   src/resources/Agent/Actor/Group/GroupShow.js
	new file:   src/resources/Agent/Actor/Group/GroupTitle.js
	new file:   src/resources/Agent/Actor/Group/index.js
	modified:   src/resources/Agent/Agent.js
	modified:   src/resources/index.js

capucinemasson committed:
modified:   package-lock.json
	modified:   src/resources/Agent/Actor/Actor.js
	new file:   src/resources/Agent/Actor/Group/GroupEdit.js
	new file:   src/resources/Agent/Actor/Group/GroupList.js
	new file:   src/resources/Agent/Actor/Group/GroupShow.js
	new file:   src/resources/Agent/Actor/Group/GroupTitle.js
	new file:   src/resources/Agent/Actor/Group/index.js
	modified:   src/resources/Agent/Agent.js
	modified:   src/resources/index.js

capucinemasson committed:
Automatic prettier

srosset81 commented on src/frontend/boilerplates/archipelago/src/resources/Agent/Actor/Group/GroupList.js:
```diff
@@ -0,0 +1,29 @@
+import ListIcon from '@material-ui/icons/List';
+
+const GroupList = props => (
+  <MultiViewsList
```
Comme il n'y a qu'une vue, le composant MultiViewList n'est pas nécessaire. Vous pouvez utiliser le composant List (voir comment c'est fait pour ThemeList, par exemple).

srosset81 commented on src/frontend/boilerplates/archipelago/src/resources/Agent/Actor/Group/GroupShow.js:
```diff
@@ -0,0 +1,53 @@
+      <Column xs={12} sm={3} showLabel>
+        <UriArrayField reference="Person" source="pair:affiliates">
+          <GridList xs={6} linkType="show">
+            <UserIcon />
```
UserIcon est deprecated, merci d'utiliser AvatarField à la place (voir SkillShow par exemple)

srosset81 commented on src/frontend/boilerplates/archipelago/src/resources/Agent/Actor/Group/index.js:
```diff
@@ -0,0 +1,40 @@
+  },
+  dataModel: {
+    types: ['pair:Group'],
+    containerUri: process.env.REACT_APP_MIDDLEWARE_URL + 'groups',
```
Il faudrait aussi ajouter le container /groups dans ce fichier:
https://github.com/assemblee-virtuelle/semapps/blob/master/src/middleware/boilerplates/pair-ldp-server/containers.js

srosset81 reviewed (changes requested):
Merci @capucinemasson @mathildesauvage !
Quelques petits ajustements ci-dessous mais c'est presque bon.

capucinemasson committed:
Update containers.js

capucinemasson committed:
modified:   frontend/boilerplates/archipelago/src/resources/Agent/Actor/Group/GroupList.js
	modified:   frontend/boilerplates/archipelago/src/resources/Agent/Actor/Group/GroupShow.js

capucinemasson committed:
Merge branch 'groupe_to_archipel_project' of github.com:assemblee-virtuelle/semapps into groupe_to_archipel_project

srosset81 committed:
Merge branch 'master' into groupe_to_archipel_project

# Conflicts:
#	src/frontend/boilerplates/archipelago/src/resources/Agent/Actor/Person/PersonShow.js
#	src/middleware/boilerplates/pair-ldp-server/.env
#	src/middleware/boilerplates/pair-ldp-server/package-lock.json

srosset81 committed:
Add missing import

srosset81 committed:
Automatic prettier

srosset81 commented:
C'est parfait, merci ! :-)
J'ai fixé les conflits de merge et ajouté un import manquant. Je vais merger et déployer sur Archipel.
